packages/icons-react/README.md

```diff
@@ -35,11 +35,102 @@ respectively.
 To import using CommonJS, you can do the following:
 
 ```js
-const Add24 = require('@carbon/icons-react/lib/Add/24');
+const { Add24 } = require('@carbon/icons-react');
 ```
 
 _Note: if you would like to find the import path for an icon, you can reference
-our [icon preview](https://carbon-elements.netlify.com/icons/examples/preview/)_
+our
+[Icon Library](https://www.carbondesignsystem.com/guidelines/iconography/library)_
+
+### Icon fill
+
+All icons from the library support being styled by the `fill` property. You can
+change the color of an icon by passing in a custom class name that sets this
+property (preferred), or by passing in an inline style. For example:
+
+```css
+// CSS custom class name to set the fill of the icon to `rebeccapurple`
+svg.my-custom-class {
```

Comment:
great color :)

The element selector might be redundant? 
```suggestion
.my-custom-class {
```

Reply:
If someone wants to override it down the line they’d have to also use the element selector

Reply:
@vpicone to be honest it's just the intent is to make it clear that this targets the `svg` node since there is a layer of indirection here by using the icon module itself.

Reply:
tbh i think i like the `svg.*` notation as well. if a component has nested divs/etc i always appreciate a css rule that points out which rules are for the unique/important elements

Reply:
Any future rules meant to target fill (hover, secondary variations) will all have to also use the element due to the extra specificity. 

Future developers might not expect the extra specificity and get frustrated that their changes aren't being applied. The fill property only applies to svg's so signaling the element type here is unnecessary in my opinion.

Reply:
While it wouldn't make a difference here, depending on the element type introduces a dependency on the mark up could which could to be come more semantic or more general and break all of your styles.

Reply:
> The fill property only applies to svg's so signaling the element type here is unnecessary in my opinion.

There is the root `svg` that this can apply to, alongside any descendant of an `svg` that accepts fill which is why this specificity is important. Otherwise, it could be directed towards underlying `path`s, `circle`, etc.